src/types.ts

```diff
@@ -1483,6 +1484,7 @@ export type MessageBase<
   id: string;
   pinned: boolean;
   attachments?: Attachment<AttachmentType>[];
+  cid?: string;
```

Comment:
isn't this part of the MessageResponse type? we don't expect the user to set this